FirebaseCombineSwift/Tests/Unit/Functions/HTTPSCallableTests.swift

```diff
@@ -36,12 +38,16 @@ class HTTPSCallableTests: XCTestCase {
   func testCallWithoutParametersSuccess() {
     // given
     var cancellables = Set<AnyCancellable>()
-    let funcationWasCalledExpectation = expectation(description: "Function was called")
+    let functionWasCalledExpectation = expectation(description: "Function was called")
+    functionWasCalledExpectation.assertForOverFulfill = true
```

Comment:
If I'm not mistaken the expectation has [assertForOverFulfill == true](https://developer.apple.com/documentation/xctest/xctestexpectation/2806575-assertforoverfulfill) is `true` and [expectedFulfillmentCount == 1](https://developer.apple.com/documentation/xctest/xctestexpectation/2806572-expectedfulfillmentcount) by default here, so the line may be removed.

Reply:
You're right, thanks for pointing this out.